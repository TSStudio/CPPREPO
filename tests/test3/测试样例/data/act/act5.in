6
aa 1
bb 4
cc 7
dd 3
ee 5
ff 11