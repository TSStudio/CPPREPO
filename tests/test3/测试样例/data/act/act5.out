aa
dd
bb
ee
cc
ff